### ISSUE TITLE
plans to host on dockerhub?

### ISSUE THREAD

**Issue by 0xaf1f** [ssadedin/ximmer#7]
You have instructions to build the Docker container, but I'd like to convert it into a Singularity container. The easiest way to do this would involve the Docker container living on a registry like DockerHub. Would you mind putting it up there?

Thanks for your consideration

**Comment by ssadedin** [comment]
I'm more than happy to do that. I haven't done it before so not sure if there are any technical challenges involved, but assuming it's straight forward, I would love to have it hosted there.

**Comment by 0xaf1f** [comment]
Yes, it's pretty straightforward. You just make an account there, link it to your github account, and set up an auto-build based on this repository. link your github repository, tell it where to find the Dockerfile, and it will build on every update.

**Comment by apeltzer** [comment]
+1 for this. That would also make it possible for others to simply pull the image without having to build it themselves, e.g. in a cluster environment / using Singularity images converted from that docker image.

**Comment by 0xaf1f** [comment]
For the record here, I had to modify the Dockerfile to be able to properly convert it to Singularity. I've made the PR https://github.com/ssadedin/ximmer/pull/8.

**Comment by hkrzystek** [comment]
Hi, I want to do the same: convert the Docker container to Singularity so I can use Ximmer on our research cluster (docker is not installed as a module, but singularity is). What's the status on this, is this Docker container available on DockerHub?
